components/Article/PayNote.js

```diff
@@ -132,44 +154,49 @@ const getPayNoteVariation = ({
     }
   }
   return isEligibleForTrial && (inNativeIOSApp || trial)
-    ? getTryVariation(seed, { query })
-    : getBuyVariation(seed, { query, isSeries })
+    ? getTryNote(seed, {
+        query,
+        customNotes: getTryNotes(customPayNotes)
+      })
+    : getBuyNote(seed, {
+        isSeries,
+        customNotes: getBuyNotes(customPayNotes)
+      })
 }
 
 const Translation = compose(
   withT,
   graphql(memberShipQuery)
-)(({ t, data: { membershipStats }, baseKey, position, element }) => {
-  const tKey = [baseKey, position, element].filter(el => el).join('/')
+)(({ t, data: { membershipStats }, payNote, position, element }) => {
+  const key = payNote.key
+    ? [payNote.key, position, element].filter(el => el).join('/')
+    : `${position}${capitalize(element) || 'Body'}`
   const count = countFormat((membershipStats && membershipStats.count) || 20000)
+  const missingKey = payNote.payNote ? payNote.payNote[key] : ''
   return (
     <RawHtml
       dangerouslySetInnerHTML={{
-        __html: t(tKey, { count: count }, '')
+        __html: t(key, { count: count }, missingKey)
```

Comment:
Should work too and be more explicit, maybe even add a comment that we just use `t` to do the count replacement:
```suggestion
        __html: t(customText ? false : key, { count }, customText)
```

Alternatively since it's only one replacement, might be clearer like this:
```suggestion
        __html: customText
          ? customText.replace('{count}', count)
          : t(key, { count })
```
(the t formatter also just does a single replacement)